stratum/hal/lib/common/utils.cc

```diff
@@ -478,5 +479,12 @@ std::string ConvertLogSeverityToString(const LoggingConfig& logging_config) {
   }
 }
 
+bool IsRegularFile(const std::string& filename) {
```

Comment:
The Google coding guidelines (which Stratum follows) use two spaces for simple indentation.

Reply:
Fixed